subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3128,12 +3128,37 @@ static inline void event_len_prep(struct ll_conn *conn)
 		struct pdu_data_llctrl_length_req *lr;
 		struct pdu_data *pdu_ctrl_tx;
 		struct node_tx *tx;
+		/*
+		 * Using bool instead of u8_t increases code size
+		 * in this case
```

Comment:
Nitpick, missing a full stop at the end.

Reply:
Nevertheless fixed.